app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,34 @@
+var ready;
+ready = function() {
+
+  $('.comment_content').keypress(function(event) {
+    if (event.which == 13) {
+        event.preventDefault();
+        event.stopPropagation();
+        var self = this;
+        $.post(this.form.action, $(this.form).serialize(), function(response){
+          clearCommentForm(self);
+          updateComments(response, self);
+        });
+    }
+  });
+
+  function updateComments(comments, self) {
+    var articleId = $(self).closest("article").attr("id");
+    var ul = 'article#' + articleId +' ul.comments ';
+    $(ul + 'li:not(:first)').remove();
+    var list = $(ul);
+    for (var i in comments) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.